packages/contracts/contracts/optimistic-ethereum/chain/CanonicalTransactionChain.sol

```diff
@@ -29,9 +29,11 @@ contract CanonicalTransactionChain is ContractResolver {
 
     address public sequencer;
     uint public forceInclusionPeriodSeconds;
+    uint public forceInclusionPeriodBlocks;
     uint public cumulativeNumElements;
     bytes32[] public batches;
     uint public lastOVMTimestamp;
+    uint public lastOVMBlocknumber;
```

Comment:
Any reason not to capitalize `Number`?

Reply:
Ugh yeah you're right, I did lowercases all over the place but that's non consistent with rest of the repo, will update!